internal/pkg/agent/cmd/run.go

```diff
@@ -501,6 +501,9 @@ func tryDelayEnroll(ctx context.Context, logger *logger.Logger, cfg *configurati
 	}
 	options.DelayEnroll = false
 	options.FleetServer.SpawnAgent = false
+	// enrollCmd daemonReloadWithBackoff is broken, setting
+	// SkipDaemonRestart to true avoids running that code.
```

Comment:
This is a bit vague, how is it broken?

Reply:
made an issue and added link to it in the comment.